test/README

```diff
@@ -25,6 +25,15 @@ expects to find it (do NOT run the build step as root):
 
    $ ./test/verify/testsuite-prepare
 
+If you want to test already existing cockpit rpm or deb packages (and
+potentially others like storaged), put them into test/tmp/build-results/ and
```

Comment:
Heh, right, udisks 2.6.4 is out, so storaged is soon a thing of the past. As this is a bit confusing, and in no way specific to storaged/udisks, I just changed it to "docker".